client/blocks/support-article-dialog/README.md

```diff
@@ -0,0 +1,30 @@
+Support Article Dialog
+=========
+
+The `SupportArticleDialog` component can be used to display render a support article right in Calypso, getting useful content in front of folks without the need to link away from the app - giving a better and faster experience!
```

Comment:
Not sure if "display render" is intentional? You could also use the active voice here, like 

> The `SupportArticleDialog` component displays a support article right in Calypso, enabling you to get useful content...

or 

> You can use the `SupportArticleDialog` component to display a support article right in Calypso., getting useful content...

If I understand it right, the component is showing a post from WP site `support.wordpress.com`. It might help people like me if we make it clear the `postId` is the ID of a post on that particular site.